### ISSUE TITLE
Move API key from query string into Authorization header

### ISSUE THREAD

**Issue by callumdmay** [Hey-Mentor/apis#9]
After doing a little research into REST models and stuff we might want to update our request structure. We should move the api key (token) into the Authorization header, as having it in the url exposes it to being logged in the url as plaintext in any request logs.

**Comment by callumdmay** [comment]
@bongiovimatthew I want to do this at some point now that we have morgan logging the http requests, would you be up for putting in a parallel PR in the mobile repo to make sure we don't break anything?

**Comment by bongiovimatthew** [comment]
Yeah, that should be a pretty small client-side change. I can take a look at this sometime this week.
